src/contract/ContractCreateTransaction.js

```diff
@@ -237,6 +248,24 @@ export default class ContractCreateTransaction extends Transaction {
         return this;
     }
 
+    /**
+     * @returns {?Uint8Array}
+     */
+    get initcode() {
+        return this._initcode;
+    }
+
+    /**
+     * @param {Uint8Array} initcode
+     * @returns {this}
+     */
+    setInitcode(initcode) {
+        this._requireNotFrozen();
+        this._initcode = initcode;
```

Comment:
Q: Does this require some kind of OneOf check to ensure either `initCode` or `fileID`) is set and not both

Reply:
I believe this is handled by `protobufjs`. If you set both fields it only encodes one. Perhaps we should set the the other field to null manually, but that may not be obvious to the users?